cmd/influxd/launcher/storage_test.go

```diff
@@ -255,6 +256,226 @@ func TestLauncher_DeleteWithPredicate(t *testing.T) {
 	}
 }
 
+func TestLauncher_FluxCardinality(t *testing.T) {
+	l := launcher.RunAndSetupNewLauncherOrFail(ctx, t)
+	defer l.ShutdownOrFail(t, ctx)
+
+	// Run a query without any data on the server - should return 0 and not crash.
+	query := `import "influxdata/influxdb"
+	influxdb.cardinality(
+		bucket: "BUCKET",
+		start: 2000-01-01T00:00:00Z,
+		stop: 2000-01-02T00:00:00Z,
+		predicate: (r) => true
+	)`
+
+	exp := `,result,table,_value` + "\r\n" +
+		`,_result,0,0` + "\r\n\r\n"
+
+	buf, err := http.SimpleQuery(l.URL(), query, l.Org.Name, l.Auth.Token)
+	if err != nil {
+		t.Fatalf("unexpected error querying server: %v", err)
```

Comment:
Could we use `require.NoError` instead of `t.Fatal` in these spots? The test is mixing both styles now

Reply:
Updated! Was some cruft from copy/paste of previous tests. Using `require` makes the error messages more readable as well IMO, so that's a nice bonus.